linode/manager
# M3-2565 Change in_progress event polling interval back to 1s

## Type of Change
- Non breaking change ('update', 'change')

## To test:

1) Kick off a Linode Resize.
2) Open the Network tab in Chrome Dev Tools. 
3) Once the Resize begins, requests to `/events` should be happening every second.
4) Once the Resize is complete, requests to `/events` should resume the normal polling sequence (1s, 2s, 4s, etc.)
